ckcp/docker/Dockerfile

```diff
@@ -0,0 +1,35 @@
+#use golang as base image and compile kcp
+FROM golang:1.17 as src
+
+LABEL usage="using multi-stage build to compile kcp using go"
+
+WORKDIR /go/src/app
+RUN git clone https://github.com/kcp-dev/kcp.git && cd ./kcp && git checkout ae4e07fcd4399264dccab47509ef17f2851050ee
+
+WORKDIR /go/src/app/kcp
+RUN go build -ldflags "-X k8s.io/component-base/version.gitVersion=v1.22.2 -X k8s.io/component-base/version.gitCommit=5e58841cce77d4bc13713ad2b91fa0d961e69192" -o bin/kcp ./cmd/kcp
+
+FROM fedora:latest
+
+LABEL comments="Dockerfile to run kcp inside a container"
+
+#install git
+RUN dnf -y install git
```

Comment:
I don't think you need all that

Reply:
yes, was using these when I was running a single build Dockerfile. Have removed these now since I am using multi-stage builds.